### PR TITLE
coteditor: update uninstall and livecheck

### PR DESCRIPTION
**Pull request by toobuntu**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/c/coteditor.rb [hunk 1]
```diff
@@ -77,9 +77,14 @@
   app "CotEditor.app"
   binary "#{appdir}/CotEditor.app/Contents/SharedSupport/bin/cot"
 
+  uninstall quit: [
```
Comment (bevanjkay) [inline Casks/c/coteditor.rb:80]: Can you make this a string instead of a single length array?

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/c/coteditor.rb (hunk 1 above)
Comment (toobuntu) [reply Casks/c/coteditor.rb]: Done!

**Comment by bevanjkay** [comment]
There is a CI issue, ref: https://github.com/Homebrew/brew/pull/16438

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**